### PR TITLE
feat: add pagination to messages view

### PR DESCRIPTION
**Pull request by piotrziolko**

### CODE REVIEW

File: src/routes/dashboard/streams/[streamId=i32]/topics/[topicId=i32]/partitions/[partitionId=i32]/messages/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  export let data;
+  $: topic = data.topic;
+  $: partitionMessages = data.partitionMessages;
+  $: prevPage = $page.url.pathname.split('/').slice(0, 6).join('/') + '/';
+
+  let currentPage = 1;
+  $: totalPages = Math.ceil(partitionMessages.currentOffset / data.pagination.count);
+
+  async function loadPage(page: number) {
+    const offset = (page - 1) * data.pagination.count;
```
Comment (numinnex) [inline src/routes/dashboard/streams/[streamId=i32]/topics/[topicId=i32]/partitions/[partitionId=i32]/messages/+page.svelte]: What is the order in which messages are paginated ? from lowest to highest ? if so I think it's better idea to display newest messages (largest offset) on first pages, I think `const offset = (totalPageNumber - page) * data.pagination.count` does the trick ?

Comment (piotrziolko) [reply src/routes/dashboard/streams/[streamId=i32]/topics/[topicId=i32]/partitions/[partitionId=i32]/messages/+page.svelte]: The paginator now can sort messages both ways - default is descending (newest first)

**Comment by spetz** [comment]
Looks great, please try to make the timestamp column display the full datetime (including microseconds if possible, it's always UTC), and render raw base64 encoded payload, as it can be any binary data under the hood.

**Comment by spetz** [comment]
LGTM, thanks!

**Review by spetz — APPROVED** [review]

**Pull request merged**